src/docs/verbose-messaging.dox

```diff
@@ -0,0 +1,62 @@
+/*! @page verbose_messaging Verbose Messaging
+
+As part of normal operation, WiredTiger will produce verbose messages
+around various events, actions and possible issues/errors. Verbose
+messages are spread across various categories, representing different sub-systems
+throughout WiredTiger. Verbose messages can be usefully leveraged to monitor,
+debug and diagnose events across WiredTiger's runtime.
+
+@section verbosity_levels Verbosity Levels
+
+WiredTiger associates a verbosity level with each message. The verbosity
+level denotes to the severity of a given message, ranging from high
+severity error messages (\c ERROR) to low severity debug informational
+messages (\c DEBUG). The full range of verbosity levels and their associated
+meanings are:
+
+<table>
+@hrow{Level, Value, Description}
+@row{\c ERROR, -3, Critical error conditions triggered in WiredTiger. Usually requiring a DB restart. }
+@row{\c WARNING, -2, Warning conditions potentially signaling non-imminent errors and behaviors. Important to be aware of and useful for diagnosing potential error states. }
+@row{\c NOTICE, -1, Messages for significant events in WiredTiger\, usually worth noting. Notice messages are associated with normal runtime operations i.e not produced from any state considered harmful or error-prone.}
+@row{\c INFO, 0, Informational style messages\, usually describing statuses and actions performed by WiredTiger. }
+@row{\c DEBUG, 1, Low severity messages\, useful for debugging purposes. }
+</table>
+
+@section verbosity_categories Verbosity Categories
+
+Each verbose message is associated with a category. Verbose categories represent a range of different
+sub-systems in WiredTiger, where messages will denote to a specific operation or event under a given sub-system.
+To see the complete exhaustive list of available verbose categories, refer to the \c 'verbose' configuration option
+under ::wiredtiger_open.
+
+Each verbosity category can be individually configured with a verbosity level (as described in @ref verbosity_levels).
+This allowing the end user to fine tune and tailor the verbosity of each category to fit their use case
+e.g configuring WiredTiger's runtime for debugging/monitoring a specific sub-system.
+
+When a category is assigned a verbosity level, any associated message less than or equal to (\c <=) that level will be displayed.
+As an example, configuring the \c checkpoint category with the \c DEBUG verbosity level would allow any \c checkpoint related messages
+less than or equal to the \c DEBUG level be displayed.
```

Comment:
Missing "to" in `level be displayed`

Reply:
Adding `to` didn't seem to be the best fix for readability. Though I took another shot and reworded the sentence in ba911edd9bb